lib/galaxy/datatypes/converters/bed_to_gff_converter.py

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env python
 # This code exists in 2 places: ~/datatypes/converters and ~/tools/filters
-from __future__ import print_function
```

Comment:
I might need to back out tweaks like this, for tools that can still run with an older python?

Reply:
The tool XML already has `<requirement type="package" version="3.7">python</requirement>`, so I think it's fine.